### PR TITLE
feat: add Info properties to Intl.Locale main page

### PR DESCRIPTION
**Pull request by romulocintra**
Add Info properties to Intl.Locale 

Related issue : https://github.com/tc39/ecma402-mdn/issues/21

### CODE REVIEW

**Review by sideshowbarker — APPROVED** [review]
Great to see these getting added

File: files/en-us/web/javascript/reference/global_objects/intl/locale/index.html [hunk 1]
```diff
@@ -37,24 +37,35 @@ <h2 id="Instance_properties">Instance properties</h2>
...
  <dd>Returns the region of the world (usually a country) associated with the locale.</dd>
  <dt>{{jsxref("Intl/Locale/script", "Intl.Locale.prototype.script")}}</dt>
  <dd>Returns the script used for writing the particular language used in the locale.</dd>
+ <dt>{{jsxref("Intl/Locale/textInfo", "Intl.Locale.prototype.textInfo")}}</dt>
+ <dd>Returns the part the ordering of characters <code>right-to-left</code> and <code>left-to-right</code>.</dd>
+ <dt>{{jsxref("Intl/Locale/timeZones", "Intl.Locale.prototype.timeZones")}}</dt>
+ <dd>Returns an {{jsxref("Array")}} of time zone identifiers, according to a Zone or Link name of the {{InterWiki("wikipedia", "Daylight_saving_time#IANA_time_zone_database", "IANA time zone database")}}</dd>
+ <dt>{{jsxref("Intl/Locale/weekInfo", "Intl.Locale.prototype.weekInfo")}}</dt>
+ <dd>Returns <a href="https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Patterns_Week_Elements">UTS 35's Week Elements</a> according to the locale's rules.</dd>
 </dl>
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/intl/locale/index.html:68]:
```suggestion
</dl>

```

**Comment by sideshowbarker** [comment]
@romulocintra It seems this PR is still set to Draft status? Is that intentional? Do you have further changes you plan to push to this PR branch?

**Comment by romulocintra** [comment]
> @romulocintra It seems this PR is still set to Draft status? Is that intentional? Do you have further changes you plan to push to this PR branch?

Yes is intentional i have several changes pending based on tc39/ecma402-mdn#21

**Comment by sideshowbarker** [comment]
@romulocintra Any update on the status of this?

**Comment by romulocintra** [comment]
> @romulocintra Any update on the status of this?

Will start working again on it next week, hopefully, can create a legit PR

**Comment by romulocintra** [comment]
Ok, just notice few changes from html to md amending all of those in a new branch cause would be easy to reconcile with upstream. soon as I have a legit PR update this one pointing to the updated one

**Comment by romulocintra** [comment]
Close in favour of #8768  & https://github.com/mdn/content/pull/8985

**Pull request closed without merging**